Example/Example/Views/Pages/HttpPage.xaml.cs

```diff
@@ -1,17 +1,48 @@
-﻿using System;
+﻿using Example.ViewModels;
+using System;
 using System.Collections.Generic;
-
+using System.Threading;
+using System.Threading.Tasks;
 using Xamarin.Forms;
 using Xamarin.Forms.Xaml;
+using Example.Models;
+using Acr.UserDialogs;
 
 namespace Example.Views.Pages
 {
     [XamlCompilation(XamlCompilationOptions.Compile)]
     public partial class HttpPage : ContentPage
     {
+        private readonly PlaceholderUsersVM placeholderUsersVM;
+        private CancellationTokenSource cts;
+
         public HttpPage()
         {
             InitializeComponent();
+            placeholderUsersVM = new PlaceholderUsersVM();
+            BindingContext = placeholderUsersVM;
+        }
+
+        protected override void OnAppearing()
+        {
+            base.OnAppearing();
+            cts = new CancellationTokenSource();
+            Task.Run(async () => await placeholderUsersVM.Fetch());
```

Comment:
I don't have the experience to tell if this is a correct way to launch task on background.